eq-author-api/migrations/copyLegalBasisToThemes.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (questionnaire) => {
+  if (questionnaire?.legalBasis) {
+    questionnaire.themeSettings.themes.forEach(
+      (theme) => (theme.legalBasisCode = questionnaire.legalBasis)
+    );
+  }
+
+  return questionnaire;
+};
```

Comment:
Not really, no - since the only ones stored in the JSON are ones which have been enabled (at some point, even if now disabled again). Themes which haven't been enabled yet don't "exist" so won't be processed by the migration.

This also makes sure all existing `Theme`s have `legalBasisCode`s since they're now required in the graphQL response in order for the themes settings page to work (and the existing legal basis is a sensible default for that).